core/src/main/scala/filodb.core/metadata/Dataset.scala

```diff
@@ -37,6 +37,7 @@ final case class Dataset(name: String,
                          dataColumns: Seq[Column],
                          rowKeyIDs: Seq[Int],
                          downsamplers: Seq[ChunkDownsampler],
+                         hasDownsampledData: Boolean = false,
```

Comment:
To make the merge easier, can we
1) move hasDownsampledData to `DatasetOptions`?
2) Is it possible for dataColId to be moved somewhere else as well?  If it hasn't been here before it likely can be somewhere else.

Reply:
1. Done
2. Hmm... To where? When your PR introduces data schema, this would have to be moved in there.